### DISCUSSION TITLE
Shift-Space for Prev/Up

### DISCUSSION THREAD

**Discussion by Rolly-GitHub** [kickingvegas/casual-info#11] [Ideas]
The Transient menu has bound to Info-scroll-up:
\<space\>  Next/Down

Suggest you follow that item next with:
\<S-space\> Prev/Up

bound to Info-scroll-down.

Note: Maybe it would be "S-\<space\>" rather than "\<S-space\>" to represent Shift-Space.
... Not sure what the "best" or correct way to list that key sequence.

**Comment by kickingvegas** [comment]
@Rolly-GitHub Quick take: Yes this is a good suggestion. A problem though with S-SPC is that Emacs running in a TTY will not work (or at least with the terminal software I use). For this instance Emacs Info uses the DEL key to scroll back, which TBH I've always found weird. Let me see if I can explore having the menu detect if it is in GUI or TTY and offer that binding accordingly.

**Reply by Rolly-GitHub** [reply]
This is probably the easy part:

(display-graphic-p)

https://www.gnu.org/software/emacs/manual/html_node/elisp/Display-Feature-Testing.html


The hard part may be the 'do-gui-work' and 'do-tty-work' within the "transient menu" system :

```
(if (display-graphic-p)
    (do-gui-work)
  (do-tty-work))

```

**Comment by kickingvegas** [comment]
@Rolly-GitHub FYI, I've created issue #12 to make this an action item.
